source/fab/steps/archive_objects.py

```diff
@@ -49,46 +51,54 @@ def archive_objects(config: BuildConfig,
     * The object archive is a convenience step before linking a **shared object**.
     * One or more object archives as convenience steps before linking **executables**.
 
-    The benefit of creating an object archive before linking is simply to reduce the size
-    of the linker command, which might otherwise include thousands of .o files, making any error output
-    difficult to read. You don't have to use this step before linking.
-    The linker step has a default artefact getter which will work with or without this preceding step.
+    The benefit of creating an object archive before linking is simply to
+    reduce the size of the linker command, which might otherwise include
+    thousands of .o files, making any error output difficult to read. You
+    don't have to use this step before linking. The linker step has a default
+    artefact getter which will work with or without this preceding step.
 
     **Creating a Static or Shared Library:**
 
-    When building a library there is expected to be a single build target with a `None` name.
-    This typically happens when configuring the :class:`~fab.steps.analyser.Analyser` step *without* a root symbol.
-    We can assume the list of object files is the entire project source, compiled.
+    When building a library there is expected to be a single build target
+    with a `None` name. This typically happens when configuring the
+    :class:`~fab.steps.analyser.Analyser` step *without* a root symbol.
+    We can assume the list of object files is the entire project source,
+    compiled.
 
     In this case you must specify an *output_fpath*.
 
     **Creating Executables:**
 
-    When creating executables, there is expected to be one or more build targets, each with a name.
-    This typically happens when configuring the :class:`~fab.steps.analyser.Analyser` step *with* a root symbol(s).
-    We can assume each list of object files is sufficient to build each *<root_symbol>.exe*.
+    When creating executables, there is expected to be one or more build
+    targets, each with a name. This typically happens when configuring the
+    :class:`~fab.steps.analyser.Analyser` step *with* a root symbol(s).
+    We can assume each list of object files is sufficient to build each
+    *<root_symbol>.exe*.
```

Comment:
I don't know how best to deal with this in the documentation but `.exe` is a uniquely Windows concept.

Reply:
(fwiw, the .exe was there before, I just reformatted it :) ).
I've replaced all exe with executable.